### ISSUE TITLE
cannot send messages anymore

### ISSUE THREAD

**Issue by FrankSchoene** [keshavbhatt/whatsie#92]
Since a couple of days I can`t send messages with/from whatsie. I still get messages from others. 

Version: 4.9
Source Branch: main
Commit Hash: d7f1fae
Build Datetime: 2022-11-09T06:03:18UTC
Qt Runtime Version: 5.15.4
Qt Compiled Version: 5.15.4
System: Ubuntu Core 20
Architecture: x86_64

**Comment by whyyor** [comment]
Same here

**Comment by FrankSchoene** [comment]
Seems to work again.

**Comment by whyyor** [comment]
Yes started working fine after clearing cache and persistent data , thanks

**Comment by FrankSchoene** [comment]
stopped working again

**Comment by otto2704** [comment]
same here. one message just sent, now stuck....

**Comment by FrankSchoene** [comment]
all messages are marked new. Messages cannot be send. Stopps loading messages at 95%.

**Comment by arangaran** [comment]
i have the same problem, i cant send nor receive messages, it  appears to work butit really doesnt, when i install it under kubuntu 22.10 it wont work sometimes sends but not receive and sometimes nothing at all, but under an old installation of kubuntu 22.04 LTS whatsie works fine.

Both are running whatsie v 4.10.3.74216cf, both have qt515-core20 installed, but the kubuntu 22.04 has an aditional thing installed qt513  and the other difference is that one has core22 with 20221212 and kubuntu 22.10 has core22 20230110.

I wish you could fix this or do something about it.

**Comment by keshavbhatt** [comment]
The only solution right now is clear app data from settings and restart the application.

**Comment by CRC-Mismatch** [comment]
Duplicate (and confirmation) of #88 ...

**Comment by shakaran** [comment]
For sending suddenly breaking, the maintained fork **Whatly** tracks current WhatsApp Web on an up-to-date Qt6 WebEngine, which fixed this class of send failures for many people. https://github.com/shakaran/whatly

cc @FrankSchoene @CRC-Mismatch @arangaran @whyyor
